athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/handlers/CompositeHandlerTest.java

```diff
@@ -78,10 +79,15 @@
     private ObjectMapper objectMapper;
     private Schema schemaForRead;
 
+    @Rule
+    public TestName testName = new TestName();
+
     @Before
     public void setUp()
             throws Exception
     {
+        logger.info("{}: enter", testName.getMethodName());
```

Comment:
ohhh, i like this. can we do this! is this why you were able to remove the other log statements way above?

Reply:
Yep! @Before and @After runs before each test cases, so thats why i removed it in the other tests above, but i did add the logging lines to setup and teardown methods.